modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/SupportExportObjectEntriesAtInstancelevel.testcase

```diff
@@ -133,9 +133,17 @@ definition {
 
 		task ("And Given 'C_Student (v1_0 - Liferay Object REST)' as Entity Type in Data Migration Center --> Export File") {
 			ImportExport.selectEntity(entityType = "C_Student (v1_0 - Liferay Object REST)");
+
+			ImportExport.configureExport(
```

Comment:
I have added here to select the JSON as export file format, cause when CSV available it is the one that appears first on the list.

Reply:
@magjed4289  And I think if we use 'ImportExport.configureExport', the 'ImportExport.selectEntity' we don't need it anymore

Reply:
CSV is under a feature flag which is not enabled in this test. Anyway, put JSON here would be more stable when the feature flag is removed